src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Button from './Button';
+import './buttonPanel.css';
+
+export default function ButtonPanel() {
+  return (
+    <div>
+      <div className="row">
+        <Button color="yes" wide={false} name="AC" />
+        <Button color="yes" wide={false} name="+/-" />
+        <Button color="yes" wide={false} name="%" />
+        <Button wide={false} name="÷" />
+      </div>
+      <div className="row">
+        <Button color="yes" wide={false} name="7" />
+        <Button color="yes" wide={false} name="8" />
+        <Button color="yes" wide={false} name="9" />
```

Comment:
### [optional changes]
- you dont need to pass `false` value to `wide` property. Being boolean, it will default to `false`: Code below will be enough:
```
<Button color="yes"  name="7" />
<Button color="yes"  name="8" />
<Button color="yes"  name="9" />
```